src/main/java/de/dennisguse/opentracks/io/file/importer/KmzTrackImporter.java

```diff
@@ -69,20 +68,18 @@ public KmzTrackImporter(Context context, Uri uriFile) {
     public Track.Id importFile(InputStream inputStream) {
         Track.Id trackId;
 
-        if (!copyKmzImages()) {
-            cleanImport(context, importTrackId);
-            return null;
-        }
-
         try {
             trackId = findAndParseKmlFile(inputStream);
         } catch (Exception e) {
-            cleanImport(context, importTrackId);
             throw e;
```

Comment:
This try-catch is not doing anything.
Is it not necessary to cleanup the failed import on exception?

Reply:
The cleanImport needs the track id but if findAndParseKmlFile fails then trackId is not known (now the track is created after import is finishes).

Reply:
Then we still don't need to catch the exception and throw the same one - so we can remove this try catch, right?